docs/src/modules/branding/BrandingBeginToday.tsx

```diff
@@ -0,0 +1,78 @@
+import * as React from 'react';
+import Box from '@material-ui/core/Box';
+import Typography from '@material-ui/core/Typography';
+import Button from '@material-ui/core/Button';
+import Container from '@material-ui/core/Container';
+import NavigateNextIcon from '@material-ui/icons/NavigateNext';
+import Link from 'docs/src/modules/components/Link';
+
+export default function BrandingBeginToday() {
+  return (
+    <Box
+      sx={{
+        bgcolor: 'secondary.main',
+        color: 'secondary.contrastText',
+        position: 'relative',
+      }}
+    >
+      <Box
+        component="img"
+        src="/static/branding/block5.png"
+        alt=""
```

Comment:
```suggestion
        alt="Decorative blocks"
```

Reply:
I have actually removed the alt for accessibility reasons, see https://html.spec.whatwg.org/multipage/images.html#a-purely-decorative-image-that-doesn't-add-any-information. The imagine is decorative.

Reply:
Ah wasn't aware of it. Should we then add `aria-hidden` at least on these images?

Reply:
I think we should based on https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-hidden_attribute

Reply:
I could find #18745 touching on the same topic. We went with alt="" instead of aria-hidden. No idea why. @mbrookes @eps1lon What should we do here?

Reply:
Decorative images should have `alt=""`. `alt=""` signals that the image is decorative. `aria-hidden` is for things you actually want to hide. Since the image is visible to sighted users, it should also be "visible" to assistive technology. Screen readers should announce it as a decorative image if asked to.

Reply:
I think the MDN article shouldn't mention that just like you wouldn't recommend `<img role="presentation" />`: `alt=""` is more established.

Reply:
Thanks for the explanation 👍